utils/utils_test.go

```diff
@@ -53,3 +53,41 @@ func TestGetGithubBaseURL(t *testing.T) {
 		})
 	}
 }
+
+func TestIsGoFile(t *testing.T) {
+	tests := []struct {
+		got  string
+		want bool
+	}{
+		{"foo.go", true},
+		{"bar.gol", false},
+		{"foobar", false},
+		{"fubar.g", false},
+	}
+	for _, tt := range tests {
+		t.Run(fmt.Sprintf("given the filename %s", tt.got), func(t *testing.T) {
+			got := IsGoFile(tt.got)
+			assert.Equal(t, got, tt.want)
+		})
+	}
+}
+
+func TestTrimGoPath(t *testing.T) {
+	type args struct {
+		path       string
+		repository string
+	}
+	tests := []struct {
+		args args
+		got  string
+		want string
+	}{
+		// Need some test cases, but not sure what this function really does.
+	}
+	for _, tt := range tests {
+		t.Run(fmt.Sprintf("Trim GOPATH"), func(t *testing.T) {
+			got := TrimGoPath(tt.args.path, tt.args.repository)
+			assert.Equal(t, got, tt.want)
+		})
+	}
+}
```

Comment:
This is a good test dor this function.
```suggestion
func TestTrimGoPath(t *testing.T) {
	tests := []struct {
		path string
		repository string
		want string
	}{
		{fmt.Sprintf("%s/src/gocity/main.go", os.Getenv("GOPATH")), "gocity", "/main.go"},
		{fmt.Sprintf("%s/src/gocity/foo/bar.go", os.Getenv("GOPATH")), "gocity", "/foo/bar.go"},
		{fmt.Sprintf("%s/src/gocity/vendor", os.Getenv("GOPATH")), "gocity", "/vendor"},
	}
	for _, tt := range tests {
		t.Run(fmt.Sprintf("given project %s/%s", tt.path, tt.repository), func(t *testing.T) {
			got := TrimGoPath(tt.path, tt.repository)
			assert.Equal(t, tt.want,  got)
		})
	}
}
```

Reply:
Hi Rodrigo! Thanks! I see now, silly me 😅